frontend/packages/supabase/prisma/schema.prisma

```diff
@@ -0,0 +1,15 @@
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
```

Comment:
I am wondering how I should manage environment variables. We'll consider it later.